scudcloud-1.0/lib/scudcloud.py

```diff
@@ -203,6 +208,7 @@ def addMenu(self):
         viewMenu.addAction(self.menus["view"]["reset"])
         viewMenu.addSeparator()
         viewMenu.addAction(self.menus["view"]["fullscreen"])
+        viewMenu.addAction(self.menus["view"]["hidemenu"])
```

Comment:
@mbra sorry to ask you more one thing... can you add a check here to see if it's not Unity? I.e., add this entry in the menu only when environment is not Unity? (You can use something like `if Unity is not None:`).